currency/l1/public.yml

```diff
@@ -159,3 +160,133 @@ paths:
             text/html:
               schema:
                 type: string
+
+  /data/estimate-fee:
+    post:
+      tags:
+        - Data
+      description: Estimate fee for given data update.
+      requestBody:
+        description: Data update.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../../models/schemas/EstimateFee.yml'
```

Comment:
Schema here will be the same as the POST /data endpoint: SignedDataUpdate

Reply:
@AlexBrandes  @ryle-goehausen specifically requested the input be `DataUpdate`, i.e. not signed.

Reply:
Good catch - was saying the same thing in a parallel convo. Should not be signed, my mistake.